client/src/components/setup/welcome-slide.tsx

```diff
@@ -16,7 +16,7 @@ export function WelcomeSlide(props: {
   return (
     <Slide
       classes={classes}
-      title="Welcome to MentorPal!"
+      title="Welcome to MentorStudio!"
```

Comment:
Are we changing the name of the admin portal to Mentor Studio? Are there other places that also need changing in other projects?

Reply:
eah, this was per Ben’s request